src/test/java/teammates/test/cases/ui/browsertests/InstructorFeedbackPageUiTest.java

```diff
@@ -355,7 +355,7 @@ public void testAddAction() throws Exception {
         savedSession = BackDoor.getFeedbackSession(newSession.getCourseId(), newSession.getFeedbackSessionName());
         newSession.sanitizeForSaving();
 
-        newSession.setInstructions(new Text("<p>cannot see responses</p>\r\n\r\n<p>$^/&#61;?</p>"));
```

Comment:
@thyageshm That is to revert changes made to InstructorFeedbackPageUiTest.java. I'll remove it.